crates/cli/tests/wasm-bindgen/reference.rs

```diff
@@ -139,10 +139,36 @@ fn runtest_targets_atomics() -> Result<()> {
     test.push("reference");
     test.push("targets.rs");
 
-    runtest_with_opts(test, true)
+    runtest_with_opts(test, Some("atomics"), |project| {
+        project
+            .cargo_cmd
+            .env("RUSTUP_TOOLCHAIN", "nightly")
+            .env("RUSTFLAGS", "-C target-feature=+atomics")
+            .arg("-Zbuild-std=std,panic_abort");
+    })
+}
+
+#[test]
+fn runtest_targets_mvp() -> Result<()> {
+    let mut test = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    test.push("tests");
+    test.push("reference");
+    test.push("targets.rs");
+
+    runtest_with_opts(test, Some("mvp"), |project| {
+        project
+            .cargo_cmd
+            .env("RUSTUP_TOOLCHAIN", "nightly")
+            .env("RUSTFLAGS", "-C target-cpu=mvp")
+            .arg("-Zbuild-std=std,panic_abort");
+    })
 }
 
-fn runtest_with_opts(test: PathBuf, atomics: bool) -> Result<()> {
+fn runtest_with_opts(
+    test: PathBuf,
+    suffix: Option<&str>,
```

Comment:
Would be simpler to just change this to

```suggestion
    suffix: &str,
```

and use `""`, `"_atomics"`, `"_mvp"` explicitly at callsites. Removes allocation, dealing with `None` etc.

Reply:
I think it still makes sense for "suffix" to mean the whole thing together with underscore, but I don't care that strongly.